packages/ui/primitives/document-flow/missing-signature-field-dialog.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import { Button } from '@documenso/ui/primitives/button';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from '@documenso/ui/primitives/dialog';
+
+export type MissingSignatureFieldDialogProps = {
+  isOpen: boolean;
+  onOpenChange: () => void;
+};
+
+export const MissingSignatureFieldDialog = ({
+  isOpen,
+  onOpenChange,
+}: MissingSignatureFieldDialogProps) => {
+  return (
+    <Dialog open={isOpen} onOpenChange={onOpenChange}>
+      <DialogContent className="max-w-lg" position="center">
+        <DialogHeader>
+          <DialogTitle>No signature field found</DialogTitle>
+          <DialogDescription>
+            <p className="mt-2">
+              Some signers have not been assigned a signature field. Please assign a signature field
+              to each signer before proceeding.
+            </p>
+          </DialogDescription>
+        </DialogHeader>
+        <DialogFooter>
+          <div className="flex w-full flex-nowrap gap-4">
```

Comment:
I think you should remove this `div` so we can use the normal layout for footers to stay consistent

Reply:
Ok, thanks. I did it, although the buttons on the webhook modal is also like this (buttons on the left side). I guess I should update that as well?

Reply:
Yeah just use the defaults which is right side